src/app/shared/components/workshop-card/workshop-card.component.ts

```diff
@@ -65,6 +72,8 @@ export class WorkshopCardComponent implements OnInit, OnDestroy {
     private imagesService: ImagesService) { }
 
   ngOnInit(): void {
+    this.isPrivate = this.workshopData.providerOwnership !== 'Common' 
```

Comment:
wroooong
 this.workshopData.providerOwnership !== 'State'
and create an enum for it
and the naming is not correct
it should reflect an opportunity to close/open status, so isPrivate does not match. it should be something like "canChangeWorkshopStatus" (maybe you can come up with better naming)